docs/pokemon_optimizer.md

```diff
@@ -342,11 +342,12 @@ Every Pokemon not selected is candidate for transfer.
 | `mode`    | `"by_pokemon"`, `"by_family"`, `"overall"` | `"by_family"` |
 
 The `mode` define how the Pokemon Optimizer is going to apply the rule.
-- `"by_pokemon"` will apply the rule for each Pokemon.
-<br>In that mode, each Pokemon will be compared to other Pokemon of the same name.
+- `"by_pokemon"` will apply the rule for each Pokemon. That is for every types of Pokemon individually.
```

Comment:
ok, but the example would be misleading since it does not contains 2 pokemon of the same family.